Composer/packages/server/__tests__/mocks/samplebots/bot2/ComposerDialogs/Main.dialog

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Are these files generated during tests? If they are, they should be cleaned up as part of the test run as well as ignored from git. 

If they are not generated, we should move them to a `fixtures` directory instead of `mocks`.